dts/arm/nordic/nrf5340_cpunet_qkaa.dtsi

```diff
@@ -15,6 +15,10 @@
 	reg = <0x21000000 DT_SIZE_K(64)>;
 };
 
+&mpu {
+	arm,num-mpu-regions = <8>;
```

Comment:
I think the following line:
https://github.com/zephyrproject-rtos/zephyr/blob/959270a7b5acee25a8006d2537e0cfe22683baa6/dts/arm/nordic/nrf5340_cpunet.dtsi#L31
should be removed, to not create confusion.

Reply:
Yeap, sure, thanks for catching this.